docs/source/status.rst

```diff
@@ -337,18 +339,18 @@ C Stream Interface
 Third-Party Data Formats
 ========================
 
-+-----------------------------+---------+---------+-------+------------+-------+-------+-------+-------+
-| Format                      | C++     | Java    | Go    | JavaScript | C#    | Rust  | Julia | Swift |
-|                             |         |         |       |            |       |       |       |       |
-+=============================+=========+=========+=======+============+=======+=======+=======+=======+
-| Avro                        |         | R       |       |            |       |       |       |       |
-+-----------------------------+---------+---------+-------+------------+-------+-------+-------+-------+
-| CSV                         | R/W     | R (2)   | R/W   |            |       | R/W   | R/W   |       |
-+-----------------------------+---------+---------+-------+------------+-------+-------+-------+-------+
-| ORC                         | R/W     | R (1)   |       |            |       |       |       |       |
-+-----------------------------+---------+---------+-------+------------+-------+-------+-------+-------+
-| Parquet                     | R/W     | R (2)   | R/W   |            |       | R/W   |       |       |
-+-----------------------------+---------+---------+-------+------------+-------+-------+-------+-------+
++-----------------------------+---------+---------+-------+----+-------+-------+-------+-------+
+| Format                      | C++     | Java    | Go    | JS | C#    | Rust  | Julia | Swift |
+|                             |         |         |       |    |       |       |       |       |
++=============================+=========+=========+=======+====+=======+=======+=======+=======+
+| Avro                        |         | R       |       |            |       |       |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+-------+
+| CSV                         | R/W     | R (2)   | R/W   |            |       | R/W   | R/W   |
++-----------------------------+---------+---------+-------+------------+-------+-------+-------+
+| ORC                         | R/W     | R (1)   |       |            |       |       |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+-------+
+| Parquet                     | R/W     | R (2)   | R/W   |            |       | R/W   |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+-------+
```

Comment:
I think this is still not exactly right. We need the vertical line to continue all the way down, right?
```suggestion
+=============================+=========+=========+=======+====+=======+=======+=======+=======+
| Avro                        |         | R       |       |    |       |       |       |       |
+-----------------------------+---------+---------+-------+----+-------+-------+-------+-------+
| CSV                         | R/W     | R (2)   | R/W   |    |       |       | R/W   | R/W   |
+-----------------------------+---------+---------+-------+----+-------+-------+-------+-------+
| ORC                         | R/W     | R (1)   |       |    |       |       |       |       |
+-----------------------------+---------+---------+-------+----+-------+-------+-------+-------+
| Parquet                     | R/W     | R (2)   | R/W   |    |       |       | R/W   |       |
+-----------------------------+---------+---------+-------+----+-------+-------+-------+-------+
```